### PR TITLE
Utilize FileAbsenceCheck to force the loabalancer to remove a host

### PR DESCRIPTION
**Pull request by carolyncole**
refs #1984

### CODE REVIEW

File: config/initializers/health_monitor.rb [hunk 1]
```diff
@@ -1,20 +1,26 @@
 # frozen_string_literal: true
-HealthMonitor.configure do |config|
-  config.cache
-  config.redis
 
-  # Make this health check available at /health
-  config.path = :health
+Rails.application.config.to_prepare do
```
Comment (carolyncole) [inline config/initializers/health_monitor.rb:3]: This is explained: https://guides.rubyonrails.org/autoloading_and_reloading_constants.html#use-case-1-during-boot-load-reloadable-code

File: app/models/file_check_failure.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+class FileCheckFailure < HealthMonitor::Providers::Base
```
Comment (carolyncole) [inline app/models/file_check_failure.rb:2]: FileAbsenceCheck

File: app/models/file_check_failure.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  CHECK_FILE = "tmp/remove-from-nginx"
```
Comment (carolyncole) [inline app/models/file_check_failure.rb:3]: Make the file configurable

Comment (carolyncole) [reply app/models/file_check_failure.rb:3]: Needs to be in public/remove-from-nginx to survive deploys

File: config/deploy.rb [hunk 4]
```diff
@@ -79,6 +79,28 @@
       end
     end
   end
+
+  # You can/ should apply this command to a single host
+  # cap --hosts=pdc-describe-staging1.princeton.edu staging application:remove_from_nginx
```
Comment (maxkadel) [inline config/deploy.rb:84]: Would it be worthwhile/possible to add a check that the host parameter has been passed (like we do with the nginx playbook) to make sure you don't accidentally take both/all machines down?

Comment (carolyncole) [reply config/deploy.rb:84]: Is it not ok to take everything off line? I guess I would agree I would not normally want to... I'm not entirely sure how to do it, but I can look into it...

Comment (maxkadel) [reply config/deploy.rb:84]: Just a thought, definitely doesn't need to be a blocker!

**Comment by carolyncole** [comment]
refs https://github.com/lbeder/health-monitor-rails/pull/127

**Review by bess — APPROVED** [review]

**Pull request merged**